### PR TITLE
React routes, and Upload components added.

### PR DESCRIPTION
**Pull request by aaminaminur**
Hi Team, kindly review the React Routes and Upload component part.
Suggest the necessary upgrades if any.

### CODE REVIEW

File: client/src/Components/HomePage/HomePage.js [hunk 1]
```diff
@@ -1,12 +1,19 @@
-import logo from "../../logo.svg";
 import "./HomePage.css";
 import LandingPage from "../LandingPage/LandingPage";
-
+import { Route, BrowserRouter as Router, Switch } from "react-router-dom";
+import Navbar from "../navBar/header";
+import CreatePost from "../CreatePost/CreatePost";
```
Comment (jasce) [inline client/src/Components/HomePage/HomePage.js]: use snake case while naming your folders and files

File: client/src/Components/HomePage/HomePage.js [hunk 2]
```diff
@@ -1,12 +1,19 @@
-import logo from "../../logo.svg";
 import "./HomePage.css";
 import LandingPage from "../LandingPage/LandingPage";
```
Comment (jasce) [inline client/src/Components/HomePage/HomePage.js]: use snake case while naming your folders and files

File: client/src/Components/HomePage/HomePage.js [hunk 3]
```diff
@@ -1,12 +1,19 @@
-import logo from "../../logo.svg";
 import "./HomePage.css";
```
Comment (jasce) [inline client/src/Components/HomePage/HomePage.js]: use snake case while naming your folders and files

File: client/src/Components/CreatePost/CreatePost.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React, { Component } from "react";
+import AddPost from "./AddPost/AddPost";
```
Comment (jasce) [inline client/src/Components/CreatePost/CreatePost.js]: use snake case while naming your folders and files

File: client/src/Components/CreatePost/AddPost/AddPost.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "./AddPost.css";
```
Comment (jasce) [inline client/src/Components/CreatePost/AddPost/AddPost.js]: use snake case while naming your folders and files

File: client/src/Components/CreatePost/AddPost/AddPost.css [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+	border: solid lightgray;
+	outline: none;
+	width: 99%;
+	border-radius: 5px;
+	padding-top: 1%;
+	padding-bottom: 1%;
+	font: normal normal normal 18px/22px Verdana;
+}
+
+.PostButton {
```
Comment (jasce) [inline client/src/Components/CreatePost/AddPost/AddPost.css]: use camelCase for css classes

File: client/src/Components/CreatePost/AddPost/AddPost.css [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+.part4 {
+	display: flex;
+	padding-top: 4%;
+	padding-bottom: 4%;
+	text-align: center;
+	justify-content: center;
+	align-items: center;
+}
+
+.BrowseButton {
```
Comment (jasce) [inline client/src/Components/CreatePost/AddPost/AddPost.css]: use camelCase

File: client/src/Components/CreatePost/AddPost/AddPost.css [hunk 8]
```diff
@@ -0,0 +1,101 @@
+.BoxComponents {
```
Comment (jasce) [inline client/src/Components/CreatePost/AddPost/AddPost.css]: use camelCase

**Review by jasce — CHANGES_REQUESTED** [review]

File: client/src/Components/HomePage/HomePage.js (hunk 1 above)
Comment (aaminaminur) [reply client/src/Components/HomePage/HomePage.js]: @jasce sir, I made all changes as per you guidance, how to update it?

Comment (jasce) [reply client/src/Components/HomePage/HomePage.js]: have you renamed the directories ?

**Review by prabal01 — APPROVED** [review]
Everything looks good

**Review by jasce — APPROVED** [review]

**Pull request merged**